content/200-orm/200-prisma-client/100-queries/100-query-optimization-performance.mdx

```diff
@@ -9,6 +9,12 @@ tocDepth: 3
 
 This guide describes ways to optimize query performance, debug performance issues, and how to tackle common performance issues such as the [n+1 problem](#solving-the-n1-problem).
 
+:::note
+
+You can also use [Prisma Optimize](/optimize) to debug slow queries and follow the [recommendations](/optimize/recommendations) provided to improve query performance in your app.
```

Comment:
You can also use [Prisma Optimize](/optimize) to debug slow queries and follow the [recommendations](/optimize/recommendations) provided to improve query performance in your app.